packages/cli/src/commands/init.ts

```diff
@@ -123,6 +124,82 @@ function generateSchemasFrom(ast: AST, folder: string): void {
   logInfo(`:sparkles: Models are generated successfully!`);
 }
 
+function getMetricPluginList(): string[] {
+  const pkg = fs.readFileSync('package.json', 'utf8');
+  const deps = JSON.parse(pkg).dependencies;
+
+  const providers: string[] = [];
+  if (deps != null) {
+    for (const providerPackage of Object.values(PluginPackage)) {
+      if (providerPackage in deps) {
+        providers.push(`${PluginPackage[providerPackage]}` + 'Plugin');
+      }
+    }
+  }
+  logVerbose(
+    `Parsed package.json file and found ${providers.length} compatible plugins.`
+  );
+  return providers;
+}
+
+function generateSyftTS(metricPlugins: string[], force: boolean): void {
+  if (metricPlugins.length > 0) {
+    logVerbose(`Generating syft.ts with ${metricPlugins.join(', ')} plugin(s)`);
+  } else {
+    logVerbose('No compatible plugins found. Not generating syft.ts');
+    return;
+  }
+
+  const destinationPath = path.join('src', 'syft.ts');
+  if (fs.existsSync(destinationPath) && !force) {
+    logVerbose(`${destinationPath} already exists. Use --force to overwrite.`);
+    return;
+  }
+
+  const imports =
+    metricPlugins.length === 1
+      ? ` ${metricPlugins[0]} `
+      : '\n  ' + metricPlugins.join(',\n  ') + '\n';
+  const instantiations =
+    metricPlugins.length === 1
+      ? `new ${metricPlugins[0]}()`
+      : '\n' +
+        metricPlugins.map((input: string) => `    new ${input}()`).join(',\n') +
```

Comment:
nit: wondering if we can trigger an eslint to re-format the code. generating 4-spaces may not be the right thing.  may be add a TODO note ?

Reply:
https://linear.app/syft/issue/SYF-71/use-eslint-on-to-reformat-syftts-on-init